build.sc

```diff
@@ -39,9 +39,10 @@ object scalanet extends ScalaModule with PublishModule {
   )
 
   override def ivyDeps = Agg(
-    ivy"io.monix::monix:3.0.0",
+    ivy"io.monix::monix:3.1.0",
```

Comment:
As we are updating deps, whay you think about updating also netty as we are using some ancient version ?

Reply:
Sure, shall we do it as another PR?

Reply:
Do you want to update to monix 3.2.2? I guess Midnight depends on a specific release of scalanet, so there won't be any sudden deprecation warnings there, but I don't know who else might use this. I think it would make sense.

Reply:
Midnight depends on scalanet `0.3` which i have released last week, so its safe. From what I talked with Jeremy atala do not use scalanet, so i would say lets go for it.

Reply:
Sweet, done.